### PR TITLE
RNMobile - Release 1.28.2

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

Make sure that the pullquote block only shows for iOS devices.

## How has this been tested?

This can be tested using this GB Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2324

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by mchowning** [comment]
:wave: @SergioEstevao ! When I merged 1.28.1 into `master` I had to create a new branch off of the release branch and merge `master` into that branch order to get the "Build Artifacts" CI job to pass. I suspect we'll need to do the same here. Conveniently, that also cleans up the diff.

File: packages/block-library/src/index.native.js [hunk 1]
```diff
@@ -160,7 +166,7 @@ export const registerCoreBlocks = () => {
 		latestPosts,
 		devOnly( verse ),
 		cover,
-		pullquote,
+		iOSOnly( pullquote ),
```
Comment (mchowning) [inline packages/block-library/src/index.native.js:169]: We still want the pullquote block showing up on dev builds. This is entirely disabling it on Android though, right?

File: packages/block-library/src/index.native.js [hunk 2]
```diff
@@ -124,6 +129,7 @@ const registerBlock = ( block ) => {
 // eslint-disable-next-line no-undef
 const devOnly = ( block ) => ( !! __DEV__ ? block : null );
 
+const iOSOnly = ( block ) => ( Platform.OS === 'ios' ? block : null );
```
Comment (mchowning) [inline packages/block-library/src/index.native.js:132]: We still want the pullquote block showing up on dev builds, but this is entirely disabling it on Android, right?

Comment (SergioEstevao) [reply packages/block-library/src/index.native.js:132]: Good call, let me update the code

**Comment by mchowning** [comment]
Closing this in favor of https://github.com/WordPress/gutenberg/pull/22685 which uses a non-release branch with `master` merged in and the conflicts resolved.

**Pull request closed without merging**